Domain specification for ShowChallenge command

raquellenn opened the pull request:
storie in disl for show a specific challenge.


link for storie in notion:
https://www.notion.so/tokinetwork/12e66733aa6b48a9a79bff76139e4cfa?v=45d79c3db5de45cdac0bdcaec8f62b58&p=86dd983345934d9a993602ca2a0b2317&pm=s

raquellenn committed:
docs(show challenge documentation): disl storie for see a challenge

svallory commented on libs/atqr/domain/src/lib/domain-spec/stories/showChallenge.disl:
```diff
@@ -0,0 +1,9 @@
+context Challenges
+
+  💬showChallenge? :challengeId
+    if find :challegeId in database
```
You meant `find challenge with id equals :challengeId` and `return challenge`

svallory commented on libs/atqr/domain/src/lib/domain-spec/stories/showChallenge.disl:
```diff
@@ -0,0 +1,9 @@
+
+context Challenges
+
+  💬showChallenge? :challengeId
```
This should be in a ChallengeRepository

raquellenn committed:
docs(show challenge storie): refactoring the function

raquellenn committed:
docs(show challenge disl): put a service in the repository

svallory committed:
docs(domain-spec): :necktie: add specification for GetChallenge command

Implements the domain specification in disl for user story 0.3 [Show challenge](https://www.notion.so/tokinetwork/0-3-Ver-desafio-86dd983345934d9a993602ca2a0b2317)
